pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 'Using Campaign Builder'
+media_order: 'contact-sources.png,events.png,send-email-delay.png,send-email-delay-nonaction.png,campaign-decisions.gif'
+published: true
+taxonomy:
+    category:
+        - docs
+slug: using-campaign-builder
+twitterenable: true
+twittercardoptions: summary
+articleenabled: false
+orgaenabled: false
+orga:
+    ratingValue: 2.5
+orgaratingenabled: false
+personenabled: false
+facebookenable: true
+---
+
+---------------------
+
+The Mautic campaign builder gives you a blank canvas upon which to build your campaign workflow. The overall interface is clean and simple with easy to use events. These actions, decisions, or conditions can be added through clicking the "anchors" of events.
+
+### Sources 
+
+The first thing to be selected is where the campaign will pull in contacts from or contact sources. There are currently two options for contact sources: segments and forms. One or both can be added to the campaign.
+ 
+![](contact-sources.png)
+
+After selecting one or more sources, the next step will be to add one or more actions (most likely), decisions and/or conditions:
+
+![](events.png)
```

Comment:
```suggestion
![Screenshot showing actions](events.png)
```

Reply:
This comment was on an older file that was merged. This file doesn't exist any longer.